Second Branch

Rishi-Mishra0704 opened the pull request:
- Changed background color
- Changed font size

franclobo reviewed (changes requested):
Hi @Rishi-Mishra0704, 

# ♻️ REQUIRED CHANGES ♻️ 

![so-close-jim-halpert](https://user-images.githubusercontent.com/58642949/211602425-bc73143a-bd68-45e6-876d-0446a9eede53.gif)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] This is a second branch, what happened with the first one? Please 🙏🏽 use a descriptive name for your beach e.g. `hello-world`

- [ ] Kindly document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). Pull request should have a descriptive title 👇🏽 

| Expected | Yours |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/211603338-81ae5cbd-ce08-4e3b-8bcb-8aef75f1261f.png) | ![image](https://user-images.githubusercontent.com/58642949/211603499-8339e38e-7b31-4503-a194-fd5a0d56e87d.png) |

- [ ] There should be a commit for each file created (HTML, CSS, test), I couldn't see them in yours. Please 🙏🏽 add them and create the test.md file.

![image](https://user-images.githubusercontent.com/58642949/211604676-1d1324ef-86ca-4625-b2bc-45068f2226f6.png)

> A test.md file with the text "This file should be ignored by git".
> Make` sure that the last file is actually ignored by git and is not present in your GitHub repository.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

# **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Rishi-Mishra0704 commented:
I named the branch "second" because I thought it will  be a good way too represent second branch, since I considered the main branch to be the first one .

I have made a branch named "final_fix" since I won't be making change inside of this branch

Few things which I have updated are:

-  Changed the text from uppercase to capitalize in style.css
-  Made a test.md file
- Made sure that it was ignored by .gitignore file

brainconnect93 reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi @Rishi-Mishra0704 , 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 


<img src="https://i.pinimg.com/originals/f8/3d/cf/f83dcf0c379de146ef962251952577ab.gif">

## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
